evalml/tests/model_understanding_tests/test_feature_explanations.py

```diff
@@ -0,0 +1,369 @@
+    assert heavy == ["heavy 1", "heavy 2", "heavy 3"]
+    assert somewhat == ["somewhat 1"]
+    assert negative == []
+
+
+def test_get_influential_features_max_features_ignore_negative():
+    importance_df = pd.DataFrame(
+        {
+            "feature": ["heavy 1", "heavy 2", "heavy 3", "neg 1", "neg 2", "neg 3"],
+            "importance": [0.35, 0.3, 0.23, -0.15, -0.17, -0.43],
+        }
+    )
+    heavy, somewhat, negative = get_influential_features(importance_df, max_features=2)
+    assert heavy == ["heavy 1", "heavy 2"]
+    assert somewhat == []
+    assert negative == ["neg 1", "neg 2", "neg 3"]
+
+
+def test_get_influential_features_lower_min_importance_threshold():
+    importance_df = pd.DataFrame(
+        {"feature": np.arange(100), "importance": [0.1] * 25 + [0.05] * 75}
+    )
+    heavy, somewhat, negative = get_influential_features(importance_df)
+    assert heavy == []
+    assert somewhat == []
+    assert negative == []
+
+    heavy, somewhat, negative = get_influential_features(
+        importance_df, min_importance_threshold=0.01
+    )
+    assert heavy == []
+    assert somewhat == [0, 1, 2, 3, 4]
+    assert negative == []
+
+
+def test_get_influential_features_higher_min_importance_threshold():
+    importance_df = pd.DataFrame(
+        {"feature": np.arange(5), "importance": [0.55, 0.201, 0.15, 0.075, 0.024]}
+    )
+    heavy, somewhat, negative = get_influential_features(importance_df)
+    assert heavy == [0, 1]
+    assert somewhat == [2, 3]
+    assert negative == []
+
+    heavy, somewhat, negative = get_influential_features(
+        importance_df, min_importance_threshold=0.15
+    )
+    assert heavy == [0]
+    assert somewhat == [1, 2]
+    assert negative == []
+
+
+def test_get_influential_features_linear_importance():
+    importance_df = pd.DataFrame(
+        {
+            "feature": [
+                "heavy influence",
+                "negative influence",
+                "somewhat influence",
+                "zero influence",
+            ],
+            "importance": [0.6, -0.2, 0.1, 0.0],
+        }
+    )
+    heavy, somewhat, negative = get_influential_features(
+        importance_df, linear_importance=True
+    )
+    assert heavy == ["heavy influence", "negative influence"]
+    assert somewhat == ["somewhat influence"]
+    assert negative == []
+
+
+def test_get_influential_features_on_boundaries():
+    importance_df = pd.DataFrame(
+        {
+            "feature": ["heavy 1", "heavy 2", "heavy 3", "somewhat 1", "somewhat 2"],
+            "importance": [0.5, 0.2, 0.2, 0.05, 0.05],
+        }
+    )
+    heavy, somewhat, negative = get_influential_features(importance_df)
+    assert heavy == ["heavy 1", "heavy 2", "heavy 3"]
+    assert somewhat == ["somewhat 1", "somewhat 2"]
+    assert negative == []
+
+
+def test_readable_explanation_not_fitted(fraud_100):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+    X, y = fraud_100
```

Comment:
Extreme nit-pick, but we don't need X,y/fraud_100 here right? Since X and y default to None and we raise this ValueError :)